common/test/acceptance/pages/studio/overview.py

```diff
@@ -236,6 +236,7 @@ def subsection_expanded():
         currently_expanded = subsection_expanded()
 
         self.q(css=self._bounded_selector('.ui-toggle-expansion i')).first.click()
+        self.wait_for_element_presence(self._bounded_selector(self.ADD_BUTTON_SELECTOR), 'Subsection is expanded')
```

Comment:
I also find the method name a bit confusing. "toggle" implies you can go back and forth. But if this method simply won't work if you try to collapse, then maybe the method should not be called something with 'toggle'.

Since we're improving this test, could that be addressed as part of this PR?